apps/prairielearn/elements/pl-image-capture/pl-image-capture.css

```diff
@@ -3,3 +3,8 @@
   max-height: max(60vh, 600px);
   transition: transform 0.2s ease-in-out;
 }
+
+.qr-code-box {
```

Comment:
This class should be scoped to this element somehow to avoid potential conflicts with other elements or UI. Ideally `capture-preview` would be as well. We've usually used the element name for this, either with something like `pl-foo` on a root element or `pl-foo-*` as a prefix on individual classes. I'd lean towards the latter here.